test/bridge_test.rb

```diff
@@ -84,6 +84,11 @@
       end
     end
 
+    # mock for network-ng
+    hwinfo_items.each do |i|
+      allow(Yast::NetworkInterfaces).to receive(:devmap).with(i["dev_name"]).and_return(nil)
+    end
```

Comment:
if you need it at more places, you can do it in test helper and unstub only for specific tests.